doc/README.md

```diff
@@ -0,0 +1,10 @@
+![ROS2](../doc/images/ros2-logo.png)
```

Comment:
I think it'd be best to not copy the ros2-logo into our repo due to potential copyright and trademark issues.

Reply:
Sure, I'll check into the copyright status.